sources/osgUtil/Composer.js

```diff
@@ -247,20 +320,18 @@ define( [
         }
     };
 
-
+    // default means you do use the special optimized full screen triangle
+    // dubbed fakeFullscreenQuad
+    // no need of modelView, projection, nor texcoord
     Composer.Filter.defaultVertexShader = [
         'attribute vec3 Vertex;',
-        'attribute vec2 TexCoord0;',
         'varying vec2 FragTexCoord0;',
-        'uniform mat4 ModelViewMatrix;',
-        'uniform mat4 ProjectionMatrix;',
         'void main(void) {',
-        '  gl_Position = ProjectionMatrix * ModelViewMatrix * vec4(Vertex,1.0);',
-        '  FragTexCoord0 = TexCoord0;',
+        '  gl_Position = vec4(Vertex*2.0 - 1.0,1.0);',
```

Comment:
need a PR to export all glsl code in glsl files + a PR for splitting composer into composer.js filter.js + ilters/custom.js filters/blur.js etc